var/spack/repos/builtin/packages/icon/package.py

```diff
@@ -14,11 +14,14 @@ class Icon(AutotoolsPackage):
 
     homepage = "https://www.icon-model.org"
     url = "https://gitlab.dkrz.de/icon/icon-model/-/archive/icon-2024.01-public/icon-model-icon-2024.01-public.tar.gz"
+    git = "https://gitlab.dkrz.de/icon/icon-model.git"
+    submodules = True
 
     maintainers("skosukhin", "Try2Code")
 
     license("BSD-3-Clause", checked_by="skosukhin")
 
+    version("2025.04", tag="icon-2025.04-public")
```

Comment:
Can we _also_ add the `commit=1be2ca66ea0de149971d2e77e88a9f11c764bd22` argument?

Reply:
Why do u think it is needed? I don't see patches being added under the identical tag.

Reply:
For better reproducibility, since a tag can still be moved.